src/changelog-check.ts

```diff
@@ -160,6 +161,31 @@ async function isVersionOnlyChange(
   }
 }
 
+/**
+ * Checks if a package is marked as private in its package.json.
+ *
+ * @param repoPath - The path to the repository.
+ * @param filePath - The path to the package.json file.
```

Comment:
nit: I generally take `filePath` to be a generic path. This would be clearer:
```suggestion
 * @param packageJsonPath - The path to the package.json file.
```